### PR TITLE
Update engine environment to use numpy=1.16

### PR DESCRIPTION
**Pull request by sgfoote**
The version number updates for ooi-data and ion-functions are tentative!

### CODE REVIEW

File: anaconda_env/engine.yml [hunk 1]
```diff
@@ -4,27 +4,32 @@ channels:
 - ooi
```
Comment (tlfish3638) [inline anaconda_env/engine.yml:4]: Have you confirmed you can successfully run the following command to create a conda environment in which you can run the stream and ingest engines/handlers?
conda env create -f engine.yml

File: anaconda_build/enginemeta/meta.yaml [hunk 2]
```diff
@@ -5,12 +5,12 @@ requirements:
     run:
```
Comment (tlfish3638) [inline anaconda_build/enginemeta/meta.yaml:5]: You need to modify the enginemeta version. I would change it to 1.1.0.
Have you used this conda "recipe" to build an enginemeta package locally and then run the following command to create a conda environment you can use to run the stream and ingest engines/handlers successfully?
conda create --use-local -n engine enginemeta ingest_engine stream_engine

**Review by tlfish3638 — CHANGES_REQUESTED** [review]
The change to the enginemeta version number is easy. Verifying (if you haven't already) the other items may not be.

File: anaconda_build/enginemeta/meta.yaml (hunk 2 above)
Comment (sgfoote) [reply anaconda_build/enginemeta/meta.yaml:5]: I have updated the enginemeta version to 1.1.0 and built the enginemeta package locally. I have used that local package to create an engine conda environment and verified that it successfully runs stream_engine, ingest_engine, and the ingest-handlers. In particular, I ran a data ingest and pulled out the recently ingested data via both a sync and an async data request.

File: anaconda_env/engine.yml (hunk 1 above)
Comment (sgfoote) [reply anaconda_env/engine.yml:4]: I have built the engine conda environment via a "conda env create -f conda.yml" and verified that environment successfully runs stream_engine, ingest_engine, and the ingest-handlers (this was tested distinctly from the enginemeta bit below). In particular, I ran an ingest request (different from that used for the enginemeta test), and verified I could pull the newly ingested data out with both a sync and an async data request.

**Comment by sgfoote** [comment]
Note: my testing was performed after the package changes to engine.yml and meta.yaml included in 52150ae... commit.

**Review by tlfish3638 — APPROVED** [review]
Looks good

**Review by steinermg — APPROVED** [review]

**Pull request merged**